001-IntroToKubernetes/Coach/09-helm.md

```diff
@@ -4,12 +4,71 @@
 
 ## Notes & Guidance
 - You should deliver a demo in addition to the lecture to show what a working Helm chart looks like.
+  - Consider using the `app-languages-helm-chart` helm chart provided in the Coach Solutions folder for Challenge 9 as one that can be demoed during the lecture.  
 - You should provide guidance in the challenge as to which values should be parameterized in the Helm templates.
-- **NOTE:** Helm 3 does not create namespaces with yaml files in the chart.
-	- The YAML files in the Student Resources folder provides a YAML file that creates a whatthehack namespace. 
-	- Students can not include the namespace YAML when creating the helm chart
-	- They have to create the namespace by using the **--create-namespace** flag on the helm CLI, eg:
-		- `helm install myRelease app-languages --create-namespace whatthehack`
-- Consider using the helm chart provided in the Coach Solutions folder for Challenge 9 as one that can be demoed during the lecture.  
+	- At a minimum make sure the version label is parameterized (ie: v1, v2, etc) so that students can install different versions of the app using `helm install` easily.
+- **NOTE:** Helm 3 does not automatically create namespaces.
+	- The YAML files in the Student Resources folder provides a YAML file that creates a whatthehack namespace.
+    	- This should only be used when installing the app manually during the first part of the challenge.
+- **NOTE:** When creating their helm charts make sure that students are handling namespaces properly:
+	- Students should NOT include the namespace YAML in the helm chart
+	- The application's YAML files should NOT include a namespace, we will leave namespace creation up to helm (see below).
+
```

Comment:
I think there are a bunch of changes and additions we need to discuss for this coach's guide. To be short, the order of this guide is off and should match the order of the challenge.
1.) Explaining Helm -  I think we need to verbally script out a "hello world" demo that uses the "helm create" command to scaffold a new chart & walk the attendees through its structure/purpose.  (This work would correlate to changes we make to the Lecture deck to explain helm visually.)
2.) Deploying first helm chart LOCALLY (of the langfacts app) - Provide the helm commands needed to do this, and the commands needed to do the version update & rollback.  (you only show this as being done from ACR)
3) Publishing the helm chart to ACR is the final portion of the challenge and should be moved to the end of this guide (most of your work here would stay as is, but shifted to the end.)

Reply:
Yeah I don't get what you mean. If you don't run my commands in the exact order I have them it will not work. These are mostly dependent steps.

You're asking for way out of scope additions. Save all those great suggestions for the 001 revamp.

All I'm trying to do here is get the commands down on paper.

Reply:
Bump @jrzyshr

Reply:
Okay...  I'll restate for clarification (even though we did this on the phone!)

From my original comment...  #1) is out of scope for this PR.  I can add that later.
2 & 3 involve you just re-arranging the order of the content you have.  I'll put comments inline in the file.